hosting/src/assets/scss/example.scss

```diff
@@ -0,0 +1,3 @@
+.bg-pink {
+  background-color: pink;
```

Comment:
Lol. Please remove this let's just use default background color inherited from global styling.

Reply:
> Lol. Please remove this let's just use default background color inherited from global styling.

Will do.
This file is just for testing sass is loaded in the project or not.
This file is not loaded anywhere. Maybe in the future I'll improve the styling architecture, when I have a chance

Reply:
Done ✅